### DISCUSSION TITLE
What exactly does incognito do?

### DISCUSSION THREAD

**Discussion by adamlogan** [florisboard/florisboard#2717] [Q&A] [answered]
I am just wondering what exactly it means to have incognito enabled or disabled?

**Comment by tsiflimagas** [comment, marked as answer]
It doesn't really do something, apart from putting the icon on the background, since there's no word prediction feature yet, so the keyboard doesn't track user's writing to build up a database from which it would later make predictions. So, for now it's kind of a placeholder, till that feature is applied.
